.env.sample

```diff
@@ -2,7 +2,7 @@
 
 # Endpoints
 REPAIRS_SERVICE_API_URL=http://localdev/api/v2
-REPAIRS_SERVICE_API_TOKEN=repairsApiToken
+REPAIRS_SERVICE_API_KEY=repairsApiToken
```

Comment:
tiny tiny tint nitpick sorry 😁  But maybe change `repairsApiToken` to `repairsApiKey`
Sorry haha!

Reply:
No I agree, especially as we have another test value of token which is used in a different way